hls4ml/backends/vivado/passes/pointwise_codegen.py

```diff
@@ -0,0 +1,84 @@
+from hls4ml.model.layers import Conv1D
+from hls4ml.model.optimizer import OptimizerPass
+from hls4ml.model.types import Source
+
+
+def generate_pointwise_conv1d_fn(layer_idx, reuse_factor=1):
+    """Generate a C++ function for a pointwise convolution layer.
+
+    Args:
+        layer_idx (int): Index of layer ('index' attribute).
+        reuse_factor (int): Number of partitions to divide the input into.
+
+    Returns:
+        str: Generated C++ function
+    """
+
+    generated_code = (
+        "template<class data_T, class res_T, typename CONFIG_T>\n"
```

Comment:
Minor point, but we use single quotes, double quotes only in special cases

Reply:
done in https://github.com/jmduarte/hls4ml/pull/22/commits/d56dc7349e94ad8b6aaefe5b0f22385d3d9a9e03